components/carousel/script.js

```diff
@@ -311,7 +311,7 @@ class Carousel extends BaseComponent {
      * Move the carousel to the previous element.
      * @param {number} pageSize
      */
-    previous(pageSize) {
+    previous(pageSize = 1) {

```

Comment:
I have previously seen carousels naming this `prev` instead of `previous`.
Since this would require update on the documentation and is really a small deal, it's 👌